src/core/lv_obj.h

```diff
@@ -347,6 +350,26 @@ const lv_obj_class_t * lv_obj_get_class(const lv_obj_t * obj);
  */
 bool lv_obj_is_valid(const lv_obj_t * obj);
 
+/**
+ * Assign an id to an object if not previously assigned
+ * @param obj   pointer to an object
+ */
```

Comment:
Mention that these functions needs to implemented externally if `LV_USE_OBJID_BUILTIN` is not enabled.

Reply:
Done.